app/controllers/api/comments_controller.rb

```diff
@@ -12,6 +12,16 @@ def create
     end
   end
 
+  def index
+    if current_debate
+      comments = GetComments.new(current_debate.id).to_h
```

Comment:
To mi się nie podoba - to strzelanie z armaty do małej myszki.

Takie klasy, które są niezależne od bazy danych (w tym wypadku tworząc warstwę abstrakcji nad active recordem), a których używamy do operacji CRUD-owych, to repozytoria (możesz poczytać sobie o takim wzorcu projektowym). W tym wypadku są one trochę na wyrost, wystarczy wywołać `current_debate.comments`.

`GetComments` robi jeszcze jedną rzecz - wrapuje wszystkie komentarze, używając serializera. To można zrobić przy pomocy gema, np. `active_model_serializers` (to będzie wymagać trochę więcej pracy, żeby go użyć w innych miejscach gdzie do tej pory mieliśmy serializery) i wtedy mamy coś w stylu:

`render json: current_debate.comments, each_serializer: CommentSerializer`

Albo po prostu:

`render json: current_debate.comments.map { |comment| CommentSerializer.new(comment).to_h }`